tests/flytekit/unit/clients/test_raw.py

```diff
@@ -16,7 +15,6 @@ def test_update_project(mock_channel, mock_admin):
 
 
 @mock.patch("flytekit.clients.raw._admin_service")
-@mock.patch("flytekit.clients.raw.grpc.insecure_channel")
```

Comment:
same here,
Removed this line because we refactored `flytekit.clients.raw.py` in order to defer `import grpc` lazily.